packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.tsx

```diff
@@ -53,10 +54,10 @@ const WalletsListingRoute: React.FC = () => {
     }, [platformMapping, resetTradingPlatformActionParams, show]);
 
     useEffect(() => {
-        if (!activeWallet && firstLoginid) {
-            switchAccount(firstLoginid);
+        if (!isWalletAccountsLoading && !activeWallet && firstLoginid) {
```

Comment:
why we check for !activeWallet?

Reply:
I would think the moment accounts are loaded also activeWallet is loaded,

Reply:
We only want this useEffect to run on initial login when no active wallet is present yet, removing this check will cause the switch account to `fiatAccount || firstLoginid` to happen even when active wallet is already present:


https://github.com/binary-com/deriv-app/assets/142860499/cd6c240a-4143-4caa-b552-e429129e7106

Reply:
is it possible that activeWallet is loaded / available before or same moment as accounts list?

Reply:
I believe it is possible as changing the activeWallet check to walletAccountsList instead returns error. We'll have to look further into the order of the hooks
https://github.com/binary-com/deriv-app/assets/142860499/f62a98a4-8c5d-41b1-9a78-0d1dc93c820e